articles/azure-monitor/app/opencensus-python.md

```diff
@@ -550,6 +540,11 @@ In the list under **Active**:
 
 For more detailed information about how to use queries and logs, see [Logs in Azure Monitor](../logs/data-platform-logs.md).
 
+## Sample Applications
+
+You can find sample applications in the Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor).
```

Comment:
We already have a link for samples below. Can you change it to point to `https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor` instead of adding a new section?